client/state/rewind/size/test/reducer.js

```diff
@@ -82,4 +82,30 @@ describe( 'rewind.size reducers', () => {
 			).toEqual( 7 );
 		} );
 	} );
+	describe( 'backupsStopped', () => {
+		it( 'should return false when there is nothing in the state', () => {
```

Comment:
This scenario seems to test how the reducer will behave when the state is undefined, action type `REWIND_SIZE_SET` and also an empty size object. The current description seems a bit ambiguous as it only specifies the current state.

Maybe we could rephrase it to: `should return false when state is undefined and REWIND_SIZE_SET action is dispatched with no size object`.

What do you think?